cmd/lakectl/cmd/import.go

```diff
@@ -205,7 +201,6 @@ func init() {
 	importCmd.Flags().Bool("merge", false, "merge imported branch into target branch")
 	_ = importCmd.Flags().MarkDeprecated("merge", "import is done directly into target branch")
 	importCmd.Flags().Bool("no-progress", false, "switch off the progress output")
-	importCmd.Flags().StringP("message", "m", "Import objects", "commit message")
-	importCmd.Flags().StringSlice("meta", []string{}, "key value pair in the form of key=value")
+	withCommitFlags(importCmd, "Import objects", false)
```

Comment:
Now we allow empty messages in import also? I know it's a side effect of this PR but I'm not sure we want to add it to import and merge

Reply:
Merge already handles empty messages with a default: "Merge '%s' into '%s'" message.
I will add the same behavior for import as well